docs/alaa_review.md

```diff
@@ -0,0 +1,78 @@
+
+إضافة إلى ذلك توفر منصة كراودن(Crowdin) خاصيات أخرى مثل إمكانية مراقبة سير العمل، إمكانية إضافة ملاحظات خاصة على كل جزء يقع ترجمته من أجل أن يقرأها من سيفتح الملف لاحقا، خاصية التدقيق اللغوي لأعمال الترجمة من قبل المساهمين الآخرين و خاصية إقتراحات الترجمة و هي ليست ذات دقة عالية و لكن قد تساعد في بعض الأحيان على توفير الوقت في الكتابة أو في التفكير في طريقة صياغة الجملة،كانت هذه أبرز الإمتيازات التي ساعدتني في تسهيل و رفع مستوى الترجمة بإستعمال كراودن (Crowdin).
+
+#### `يمكنني أن ألخص أن كراودن (Crowdin) هي جيت هاب (GitHub) غير معقدة و سهلة اللإستعمال لأعمال الترجمة،`
+
+## كيف إكتشفتُ منصة *كراودن (Crowdin)* لأول مرة ؟
+
+تمَّ تعريفي على منصة كراودن (Crowdin) أول مرة خلال مساهمتي في عملية ترجمة *[كتاب منهج تورينج (The Turing Way handbook)](https://the-turing-way.netlify.app/welcome)* من اللغة الإنجليزية إلى الغة العربية و هي عملية مدعومة من قبل *[مجتمع العلوم المفتوحة في المملكة العربية السعودية (Open Science Community Saudi Arabia)](https://osc-ksa.com/)*،
+
+## نبذة عن مجتمع العلوم المفتوحة في المملكة العربية السعودية *(OSCSA)*  ![OSCSA](https://raw.githubusercontent.com/Open-Science-Community-Saudi-Arabia/localisation/main/imgs/logo.png)
+
+مجتمع العلوم المفتوحة في المملكة العربية السعودية تم إنشاؤه تماشيا مع رؤية المملكة العربية السعودية لعام 2030، والتي تُركِز على تثبيت القيم و تعزيز المعرفة وتحسين تكافؤ فرص الحصول على التعليم، يهدف مجتمع العلوم المفتوحة في المملكة العربية السعودية إلى توفير مكان يتفاعل فيه القادمين الجدد والأقران ذوي الخبرة، وإلهام بعضهم البعض لتمكين الممارسات العلمية المفتوحة والقيم في سير العمل لديهم وتعزيز السياسات التي تدعم توفير بُنية تحتية للعلوم المفتوحة، مجتمع العلوم المفتوحة في المملكة العربية السعودية هو جزء من الشبكة الدولية لمجتمعات العلوم المفتوحة والمكونة من أربعة عشر مجتمع في أوروبا،
+
+
+## نبذة عن *منهج تورينج (The Turing Way)* <img src="https://raw.githubusercontent.com/alan-turing-institute/the-turing-way/main/book/website/_build/html/_static/logo.jpg" height = "100"  style = "vertical-align: middle">
+
+منهج تورينج (The Turing Way) هو مشروع مفتوح المصدر، مفتوح للتعاون و مُدار من قبل مجتمَعه، يتضمن و يدعم مجتمعا متنوعا من المساهمين لجعل علوم البيانات متاحة، مفهومة و فعالة للجميع،  يهدف مشروع منهج تورينج لتوفير كل المعلومات التي يحتاجها الباحثين و علماء البيانات من القطاع الأكاديمي أو الصناعي أو العام لضمان سهولة جعل البحوث التي يعملون عليها قابلة لإعادة الإنتاج و الإستعمال،
+**ملاحظة:** بالنسبة لشخص ليس له إلا دراية سطحية بعلم البيانات و مجال البحث فإني من خلال قرائتي جزئا من كتاب منهج تورينج إكتشفت و تعلمت أشياءا لم أكن أدريها من قبل، إنَّ بنية الكتاب و طريقة شرحه غير معقدة و مناسبة للمبتدئين،
+
+
+## دليل المبتدئين في القيام بعمليات الترجمة بإستعمال منصة *كراودن (Crowdin)*
+
+* <p align= "right"> بعد أن تنشأ حسابا في كراودن استعمل الرابط الذي سيتم مشاركته معك من قبل أحدالمنظمين في المشروع الذي ستساهم في ترجمته و ستتم إضافة المشروع إلى قائمة مشاريعك في كراودن، بعد هذا يمكنك الولوج و ستبدو الواجهة مشابهة لهذه </p>
```

Comment:
Good job with the alignment @amj1337! Can you remove the bulletin points/numbers beacuse it shows on the left rather than the right?

Reply:
Hi @BatoolMM I solved the problem can you please check the solution ?